src/otx/algo/detection/rtdetr.py

```diff
@@ -75,13 +75,15 @@ def _customize_inputs(
         # prepare bboxes for the model
         for bb, ll in zip(entity.bboxes, entity.labels):
             # convert to cxcywh if needed
-            converted_bboxes = (
-                box_convert(bb, in_fmt="xyxy", out_fmt="cxcywh") if bb.format == BoundingBoxFormat.XYXY else bb
-            )
-            # normalize the bboxes
-            scaled_bboxes = converted_bboxes / torch.tensor(bb.canvas_size[::-1]).tile(2)[None].to(
-                converted_bboxes.device,
-            )
+            scaled_bboxes = bb
+            if len(bb):
```

Comment:
```suggestion
            if len(scaled_bboxes := bb):
```
or I think `bb` can be used as is, not assigning `scaled_bboxes`.

Reply:
Updated